### ISSUE TITLE
Improve handling of date 0000-00-00 in MySQL

### ISSUE THREAD

**Issue by kbraak** [gbif/ipt#1119]
```
What feature would like to see being added to the IPT?

Handling of the invalid date 0000-00-00 from MySQL is a very known problem, and there
are ways to create queries to transform or ignore these date cases (so it is understandable
why issue 819 was a won't fix). However, this is such a common problem that seem worth
while handling the exception in IPT to convert these occurrences to empty values (similar
to the fact that nulls are interpreted as empty when generating the CSV).
When dealing with large datasets, we learn about these occurrences only after the whole
archive processing phase finishes and since this could take a while, this becomes a
laborious process.
```

Original issue reported on code.google.com by `ammatsun` on 2014-10-17 10:28:51
